packages/client/foreman_ygg_worker/foreman_ygg_worker.spec

```diff
@@ -47,9 +47,13 @@ Worker service for yggdrasil that can act as pull client for Foreman Remote Exec
 mkdir -p _gopath/src
 ln -fs $(pwd)/src _gopath/src/%{name}-%{version}
 ln -fs $(pwd)/vendor _gopath/src/%{name}-%{version}/vendor
+ln -fs $(pwd)/go.mod _gopath/src/%{name}-%{version}/go.mod
 export GOPATH=$(pwd)/_gopath
 pushd _gopath/src/%{name}-%{version}
 %{gobuild}
+%if 0%{?rhel} > 7 || 0%{?fedora} || 0%{?suse_version}
+mv ygg_worker %{name}-%{version}
+%endif
```

Comment:
Not sure about this one. I assume this has to do with the different contents of `%{gobuild}`, resulting in differently named binaries :thinking: